cmd/importer/parser.go

```diff
@@ -237,7 +237,8 @@ func parseTable(t *table, stmt *ast.CreateTableStmt) error {
 	t.name = stmt.Table.Name.L
 	t.columns = make([]*column, 0, len(stmt.Cols))
 
-	mockTbl, err := ddl.MockTableInfo(mock.NewContext(), stmt, 1)
+	mockTbl, err := ddl.BuildTableInfoFromAST(metabuild.NewNonStrictContext(), stmt)
+	mockTbl.ID = 1
```

Comment:
what's this for?

Reply:
It seems like an old tool to import data to tidb. But I think no one uses it now...